test/suites/basic.sh

```diff
@@ -478,10 +478,6 @@ test_basic_usage() {
 
   [ "${REBOOTED}" = "true" ]
 
-  # Workaround for LXC bug which causes LXD to double-start containers
-  # on reboot
-  sleep 2
```

Comment:
What the hell was that all about? :) (just curious)

Reply:
IIRC, that was LXC still restarting the container when a post-stop hook failed

Reply:
Oh right, I think I fixed that a while ago.